config/checkstyle/checkstyle.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
```

Comment:
I think, we need not make changes to checkstyle rules if it is already commented in build.gradle file. If we would make changes to this file then it will be inherited from CCD-data-store or definition-store. For now please remove this file from PR.